sdk/storage/storage-file-datalake/swagger/README.md

```diff
@@ -132,12 +132,3 @@ directive:
     transform: >
       $.properties.Code = { "type": "string" };
 ```
-
```

Comment:
Why remove this? You simply need to replace `2020-04-08` with `2020-06-12`.

Reply:
file-datalake swagger spec uses the version "2020-06-12", we don't need to change it during the generation.

Reply:
I see.